docs/maui/views/MediaElement.md

```diff
@@ -76,11 +76,64 @@ public class MainActivity : MauiAppCompatActivity
 }
 ```
 
+Edit the `androidManifest.xml` manifest file and add the following.
+
+```csharp
+<application android:allowBackup="true" android:icon="@mipmap/appicon" android:enableOnBackInvokedCallback="true" android:supportsRtl="true">
+    <service android:name="CommunityToolkit.Maui.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
```

Comment:
Thanks @ne0rrmatrix for updating the documentation. I believe the namespace used on <service android:name should be _"CommunityToolkit.Maui.Media.Services"_ since you updated the namespace, right? If yes, please update it on the documentation.

Reply:
I updated the service name to be the same as the namespace in the PR